modules/gapi/CMakeLists.txt

```diff
@@ -178,6 +181,15 @@ ocv_module_include_directories("${CMAKE_CURRENT_LIST_DIR}/src")
 ocv_create_module()
 
 ocv_target_link_libraries(${the_module} PRIVATE ade)
+
+if(CV_TRACE AND HAVE_ITT)
+  add_definitions(-DOPENCV_WITH_ITT=1)
```

Comment:
These definitions should set only to our target I believe (and not globally)

Reply:
This part is reworked and definitions, includes and libraries linkages will be done only if we satisfy ``CV_TRACE AND HAVE_ITT`` condition